graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -0,0 +1,18 @@
+import itertools
+import random
+
+from .animals import get_animal_generators
+from .species import get_species_generators
+
+
+random.seed(0)
+
+
+if __name__ == '__main__':
+    sql_generators = [
```

Comment:
Let's move all this code into a main function, including the `random.seed(0)` call.